config/environments/development.rb

```diff
@@ -33,6 +33,12 @@
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
 
+  # Tell Action Mailer not to deliver emails to the real world.
+  # The :test delivery method accumulates sent emails in the
+  # ActionMailer::Base.deliveries array.  If SMTP used instead
+  # but without good credentials, results in 30 second pauses.
+  config.action_mailer.delivery_method = :test
```

Comment:
Not sure if the test mailer is usually used in the dev environment?

Reply:
without this we get 30 second hang times when there's an exception in dev mode (I fixed this in Tutor some time ago and just copied the fix over here).  If someone actually needs to test mailing they can change this to `:smtp` and provide some credentials.